irispy/iris_tools.py

```diff
@@ -211,10 +221,71 @@ def get_iris_response(pre_launch=False, response_file=None, response_version=Non
         del(iris_response["DATE"])
     return iris_response
 
+def fit_iris_xput(time_obs, time_cal_coeffs, cal_coeffs):
 
+    size_time_cal_coeffs = time_cal_coeffs.shape
+    size_cal_coeffs = cal_coeffs.shape
+
+    if size_time_cal_coeffs[1] != 2 or size_cal_coeffs[1]  < 2:
+
+       print('ERROR: Incorrect number of elements either in time_cal_coeffs or in cal_coeffs.')
+       return
```

Comment:
See comment on this line in PR #102.

Reply:
Edited as per suggestion given in PR #102.